### PR TITLE
add null check on  feeder1 and feeder2

### PR DESCRIPTION
**Pull request by bendaamerahm**
Signed-off-by: bendaamerahm <ahmed.bendaamer@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 1]
```diff
@@ -112,11 +112,17 @@ private static LineMapData toMapData(Line line) {
...
-                    .connectionDirection2(connectablePosition.getFeeder2().getDirection())
-                    .connectionName1(connectablePosition.getFeeder1().getName())
-                    .connectionName2(connectablePosition.getFeeder2().getName());
+            if (connectablePosition.getFeeder1() != null) {
+                builder
+                        .connectionDirection1(connectablePosition.getFeeder1().getDirection())
+                        .connectionName1(connectablePosition.getFeeder1().getName());
+            }
+
+            if (connectablePosition.getFeeder1() != null) {
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:121]: connectablePosition.getFeeder2()

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 2]
```diff
@@ -243,11 +249,17 @@ private static TwoWindingsTransformerMapData toMapData(TwoWindingsTransformer tr
...
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:258]: connectablePosition.getFeeder2()

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
We should also deal with the ConnectablePosition extension for three windings transformers ...

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java (hunk 1 above)
Comment (bendaamerahm) [reply src/main/java/org/gridsuite/network/map/NetworkMapService.java:121]: my bad

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**